File: internal/parser/testdata/sample.py
import os
from pathlib import Path

MAX_RETRIES = 3


class Config:
    def __init__(self, host: str, port: int):
        self.host = host
        self.port = port

    def address(self) -> str:
        return f"{self.host}:{self.port}"


def create_config() -> Config:
    return Config("localhost", 8080)

File: internal/parser/testdata/sample.rb
require 'json'
require_relative 'config'

MAX_RETRIES = 3

module Server
  class Config
    def initialize(host, port)
      @host = host
      @port = port
    end

    def address
      "#{@host}:#{@port}"
    end

    def self.default
      new("localhost", 8080)
    end
  end
end

def create_config
  Server::Config.new("localhost", 8080)
end
